src/components/ApartmentApplicantList/index.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
+import 'date-fns';
+import {
+  Grid,
+  Card,
+  CardHeader,
+  CardContent,
+  List,
+  ListItem,
+  ListItemText,
+  Typography,
+} from '@material-ui/core';
+import GroupAddIcon from '@material-ui/icons/GroupAdd';
+import GordonPeopleSearch from '../Header/components/PeopleSearch';
+import ApplicantListItem from './components/ApplicantListItem';
+import SaveButton from '../ApartAppSaveButton';
+import '../../views/ApartmentApp/apartmentApp.css';
+import '../../views/PeopleSearch/components/PeopleSearchResult/peopleSearchResult.css';
+
+// Create a list of applicants, displayed by name, username, and class standing.
+export default class ApplicantList extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSelection = this.handleSelection.bind(this);
+    this.handleRemove = this.handleRemove.bind(this);
+    this.handleSaveButtonClick = this.handleSaveButtonClick.bind(this);
+    this.loaderSize = 20;
+  }
+
+  handleSelection = (theChosenOne) => {
+    // Make sure the chosen username was not null
+    if (theChosenOne) {
+      // Send the selected username to the parent component
+      this.props.onSearchSubmit(theChosenOne);
+    }
+  };
+
+  handleChangePrimary = (profile) => {
+    // Make sure the chosen profile was not null
+    if (profile) {
+      // Send the selected profile to the parent component
+      this.props.onChangePrimary(profile);
+    }
+  };
+
+  handleRemove = (profile) => {
+    // Make sure the chosen profile was not null
+    if (profile) {
+      // Send the selected profile to the parent component
+      this.props.onApplicantRemove(profile);
+    }
+  };
+
+  handleSaveButtonClick = () => {
+    this.props.onSaveButtonClick();
```

Comment:
You could probably just pass this function directly as the onClick prop, rather than creating a handle method that simply calls the function.

Reply:
I either did not know that you could do that, or I couldn't get it to work with the class components. I looks like I tried that after the conversion to functional components but I could not get it to work, so I will look into that tonight most likely

Reply:
Nvm it does work. This change will be merged in soon